docs/source/models/concept_models/vivarium_mncnh_portfolio/hemoglobin_component/concept_model.rst

```diff
@@ -0,0 +1,342 @@
+.. role:: underline
+    :class: underline
+
+..
+  Section title decorators for this document:
+
+  ==============
+  Document Title
+  ==============
+
+  Section Level 1 (#.0)
+  +++++++++++++++++++++
+
+  Section Level 2 (#.#)
+  ---------------------
+
+  Section Level 3 (#.#.#)
+  ~~~~~~~~~~~~~~~~~~~~~~~
+
+  Section Level 4
+  ^^^^^^^^^^^^^^^
+
+  Section Level 5
+  '''''''''''''''
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+.. _2024_vivarium_mncnh_portfolio_hemoglobin_module:
+
+======================================
+MNCNH Portfolio: Hemoglobin Module
+======================================
+
+.. contents::
+  :local:
+  :depth: 2
+
+1.0 Overview
+++++++++++++
+
+This document is the page for the hemoglobin module of the pregnancy component
+in the MNCNH Portfolio simulation.
+
+This module will:
+
+  1. Assign starting point hemoglobin exposure based on GBD
+
+  2. Read-in necessary information from ANC module
+
+  3. Modify hemoglobin exposure according to services received during pregnancy
+
+  4. Output the following information:
+
+    - Services received during pregnancy that relate to hemoglobin (for V&V, cost counting)
+
+    - Hemoglobin exposure at the end of pregnancy for hemoglobin risk effect estimation (inputs to downstream models)
+    
+    - YLDs due to anemia throughout pregnancy and immediate postpartum (simulation result)
+
+.. todo::
+  
+  Link ANC module page when it exists
+
+.. note::
+
+  This module may have potential dependencies with the hypertension/pre-eclampsia model that have not yet been thought through. Model structure is subject to change in order to accomodate these dependencies.
+
+2.0 Module Diagram and Data
++++++++++++++++++++++++++++++++
+
+2.1 Module Diagram
+----------------------
+
+.. note::
+  
+  As discussed on the :ref:`maternal supplementation intervention document <maternal_supplementation_intervention>`, IFA and MMS have equivalent effects on hemoglobin. This diagram treats them as equivalent for this reason. However, we will need track which product was received as they have differences that will apply to other downstream modules of this simulation. 
+
+.. image:: hemoglobin_component.png
+
+2.2 Module Inputs
+---------------------
+
+.. list-table:: Hemoglobin module required inputs
+  :header-rows: 1
+
+  * - Input
+    - Source module
+    - Application
+    - Note
+  * - First trimester ANC attendance 
```

Comment:
Just noting that I don't think we currently do first/later ANC visits and so will need to update ANC approach as well.

Reply:
We currently just choose of they went to a care facility.